angular-ui/src/app/app.module.ts

```diff
@@ -26,13 +27,16 @@ import { DatePipe } from '@angular/common';
     MovieListComponent,
     MovieItemComponent,
     BrowseOptionComponent,
-
-    TimerComponent
+    UserProfileComponent
+  ],
+  imports: [
```

Comment:
I don't know which merge this happened in, but the TimerComponent, FooterComponent, and WatchHistoryComponent need to be in the declarations array above. The BrowseOptionComponent is already there. Then this is a duplicate imports array that can be deleted.

Reply:
Updated array